### DISCUSSION TITLE
Tag counts are inaccurate

### DISCUSSION THREAD

**Discussion by viharsheth** [standardnotes/forum#2750] [Suggestions]
**Describe the bug**
When looking at my tags and their counts, only the first tag on a particular note is being counted.

**To Reproduce**
1. Go to the main view
2. View the tag list on the left side of the screen
3. See error

**Expected behavior**
I would expect every tag to be counted, regardless of which order they were added to a note. So a note tagged #2021 and then #August would show a count of 1 next to #2021 and 1 next to #August. Currently it only shows 1 next to #2021 and 0 next to #August

**Screenshots**

**Desktop (please complete the following information):**
- Chrome browser

**Smartphone (please complete the following information):**
 - Device: Pixelbook
 - OS: 91.0.4472.167 
 - Browser: Chrome
 - Version: Same as above

**Additional context**

**Comment by atmoio** [comment]
I can't reproduce this. How old is your account? Can you replicate this for new notes, or only existing notes?

**Comment by viharsheth** [comment]
I figured it out. It's only counting unarchived notes. Any way to get it to
count all notes, even if unarchived? And can you show the tag count in the
folder structure?

Thanks,

Vihar

**Comment by atmoio** [comment]
Hmm, if we counted unarchived notes then the count would be misleading, since unarchived notes are hidden by default. So it would say 1, but it would show 0 notes and you'd have the opposite bug report of what we have here :)

We do have plans to add counts for Folders, once folders are a native element of the UI and not dependent on an extension.

**Comment by viharsheth** [comment]
Awesome, thanks. Looking forward to see what new features are coming.

**Comment by mind-overflow** [comment]
> We do have plans to add counts for Folders, once folders are a native element of the UI and not dependent on an extension.

@mobitar Hey! It's not completely related to this issue, although I'm really interested in the folders feature. Is there any kind of roadmap where we can see how things are progressing? I'm really looking forward to this feature. Thanks!

**Comment by JaspalSuri** [comment]
>Is there any kind of roadmap where we can see how things are progressing?

We do not have a public roadmap, primarily because estimating releases for certain features is tricky. There are various factors that affect the expected release date and we don't want to overpromise too many things.

**Comment by viharsheth** [comment]
I continue to have issues with this. Not sure if it's me or the service.
I'm using the web interface and turned off folders (having a lot of issues)
and there are a few dozen tags that now show with numbers next to each. It
seems like the counts are including unarchived, archived and trashed notes.
Isn't this the opposite of what I was told a few months ago? The tag shows
a number and there are fewer notes that appear when you click on the tag.
